pilot/pkg/networking/core/v1alpha3/cluster_test.go

```diff
@@ -1885,16 +1887,17 @@ func TestAutoMTLSClusterStrictMode(t *testing.T) {
 	g.Expect(err).NotTo(HaveOccurred())
 
 	// For port 8080, (m)TLS settings is automatically added, thus its cluster should have TLS context.
```

Comment:
is this comment outdated then? seems contradicting to the code

Reply:
slightly misleading should say socket match instead of tls context probably